pages.nl/linux/apt-get.md

```diff
@@ -0,0 +1,37 @@
+# apt-get
+
+> Hulpprogramma voor pakketbeheer van Debian en Ubuntu.
+> Zoek naar pakketten met `apt-cache`.
+> Meer informatie: <https://manpages.debian.org/latest/apt/apt-get.8.html>.
+
+- Werk de lijst van beschikbare pakketten en versies bij (het wordt aanbevolen dit uit te voeren voor elk ander `apt-get` commando):
+
+`apt-get update`
+
+- Installeer of werk het pakket bij naar de laatste beschikbare versie:
+
+`apt-get install {{package}}`
+
+- Een pakket verwijderen:
+
+`apt-get remove {{package}}`
+
+- Verwijder een pakket en de bijbehorende configuratiebestanden:
+
+`apt-get purge {{package}}`
```

Comment:
```suggestion
- Verwijder specifieke pakketten en hun configuratiebestanden:

`apt-get purge {{pakket1 pakket2 ...}}`
```

The same issue as above.
Note: translation source: Remove specific packages and their configuration files.

Reply:
Thanks for the review! Fixed in the latest commit.